src/ig/IGExporter.ts

```diff
@@ -231,90 +216,171 @@ export class IGExporter {
+    const inputIndexMarkdownPageContentPath = path.join(
+      this.igDataPath,
+      'input',
+      'pagecontent',
+      'index.md'
+    );
+    const inputIndexXMLPageContentPath = path.join(
+      this.igDataPath,
+      'input',
+      'pagecontent',
+      'index.xml'
+    );
+    const inputIndexMarkdownPagesPath = path.join(this.igDataPath, 'input', 'pages', 'index.md');
+    const inputIndexXMLPagesPath = path.join(this.igDataPath, 'input', 'pages', 'index.xml');
     let generation: ImplementationGuideDefinitionPageGeneration = 'markdown';
-    if (existsSync(inputIndexMarkdownPath)) {
+    if (existsSync(inputIndexMarkdownPageContentPath)) {
+      ensureDirSync(pageContentExportPath);
       this.copyWithWarningText(
-        inputIndexMarkdownPath,
-        path.join(igPath, 'input', 'pagecontent', 'index.md')
+        inputIndexMarkdownPageContentPath,
+        path.join(pageContentExportPath, 'index.md')
       );
-      logger.info('Copied ig-data/input/pagecontent/index.md');
-    } else if (existsSync(inputIndexXMLPath)) {
+      logger.info(`Copied ${path.join(pageContentExportPath, 'index.md')}`);
+    } else if (existsSync(inputIndexXMLPageContentPath)) {
+      ensureDirSync(pageContentExportPath);
       this.copyWithWarningText(
-        inputIndexXMLPath,
-        path.join(igPath, 'input', 'pagecontent', 'index.xml')
+        inputIndexXMLPageContentPath,
+        path.join(pageContentExportPath, 'index.xml')
       );
       generation = 'html';
-      logger.info('Copied ig-data/input/pagecontent/index.xml');
+      logger.info(`Copied ${path.join(pageContentExportPath, 'index.xml')}`);
+    } else if (existsSync(inputIndexMarkdownPagesPath)) {
+      ensureDirSync(pagesExportPath);
+      this.copyWithWarningText(inputIndexMarkdownPagesPath, path.join(pagesExportPath, 'index.md'));
+      logger.info(`Copied ${path.join(pagesExportPath, 'index.md')}`);
+    } else if (existsSync(inputIndexXMLPagesPath)) {
+      ensureDirSync(pagesExportPath);
+      this.copyWithWarningText(inputIndexXMLPagesPath, path.join(pagesExportPath, 'index.xml'));
+      generation = 'html';
+      logger.info(`Copied ${path.join(pagesExportPath, 'index.xml')}`);
     } else {
+      ensureDirSync(pageContentExportPath);
       logger.info('Generated default index.md.');
       const warning = warningBlock('<!-- index.md {% comment %}', '{% endcomment %} -->', [
         'This index.md file was generated from the "description" in package.json. To provide',
         'your own index file, create an index.md or index.xml in the ig-data/input/pagecontent',
-        'folder.',
+        'or ig-data/input/pages folder.',
         'See: https://build.fhir.org/ig/FHIR/ig-guidance/using-templates.html#root.input'
       ]);
-      const outputPath = path.join(igPath, 'input', 'pagecontent', 'index.md');
+      const outputPath = path.join(pageContentExportPath, 'index.md');
```

Comment:
I think in the spirit of not automatically generating stuff that could overwrite user content. Do we perhaps not generate an index.md by default here? This relates to some of the conversation happening in #411 about menu.xml.

Reply:
Yeah, I agree it seems to go against the new philosophy... The index.md we generate is so simple... maybe instead of generating it, we just log a warning that it didn't exist and the user should create it?  For some people, this warning will be a little overkill (if they are managing index.md elsewhere), but as a temporary solution, maybe it's not bad?

Reply:
I think we came to a solution for handling this at the end of this comment thread https://github.com/FHIR/sushi/pull/409#pullrequestreview-407642325